docs/howto-mac.md

```diff
@@ -0,0 +1,65 @@
+# RLLIB SUMO Docker environment for MacOS
```

Comment:
Would you mind adding yourself as the author of this document?

Reply:
thx for the advice, added now 1301a200792c9d667c307b0a45d21b15313c06ff